st2common/st2common/config.py

```diff
@@ -52,7 +52,12 @@ def register_opts(ignore_errors=False):
         cfg.BoolOpt('sync_remote_groups', default=False,
                     help=('True to synchronize remote groups returned by the auth backed for each '
                           'StackStorm user with local StackStorm roles based on the group to role '
-                          'mapping definition files.'))
+                          'mapping definition files.')),
+        cfg.BoolOpt('permission_isolation', default=False,
+                    help='Isolate Some resources for every user. These resources just include '
+                         'rules and executions temporarily. all resources can just be operated '
+                         'or obversed by respetive users execpt system_uer who has the highest '
+                         'authority.')
```

Comment:
Here's a slightly modified help text. With my changes, is it still accurate?
```
                    help='Isolate resources by user. For now, these resources only include '
                         'rules and executions. All resources can only be viewed or executed '
                         'by the owning user except the system_user who can view or run '
                         'everything.')
```
I always wished people would correct me more often when I was learning Portuguese as my second language. I'm happy to help or proofread any time. In this case, I would use the word "only" instead of "just" because the list of resources and actions are countable (there are two resources, and two actions).

On a technical note:
I'm not clear on what you mean by "operated or observed" - I think you're saying that you can only `view` your own rules and executions, and that you can only `re-run` your own executions? Or are you also saying that the user that created a rule is the only user who can trigger it?

Reply:
@cognifloyd Thanks for your significant pointing! I think I should use your description. What I mean in `operated or observed` refers to view or other operations( re-run exeution, trigger with a rule, etc.) from Web UI.